src/sinks/azure_monitor_logs.rs

```diff
@@ -225,8 +226,8 @@ impl AzureMonitorLogsSink {
         })
     }
 
-    fn build_request_sync(&self, events: Vec<BoxedRawValue>) -> crate::Result<Request<Vec<u8>>> {
-        let body = serde_json::to_vec(&events)?;
+    fn build_request_sync(&self, events: Vec<BoxedRawValue>) -> crate::Result<Request<Bytes>> {
+        let body = crate::serde::to_bytes(&events).unwrap().freeze();
```

Comment:
I don't love leaving the `unwrap`s here. It feels like we should either make these more explicit with an `expect` message, or make `to_bytes` more specific (e.g. operating only on `Event` or containers of them) so that we can have it not return an error in the first place.

I know this was already discussed a bit, but it also seems strange to me that there's no mention of `json` in the name of the function or its parent modules.

Reply:
Addressed in f629f97c0116bb0a20c0c1f84010936bd94af5f4 and 2162da76ced9446be0f7445a711eae97cc2fbff9.